librz/include/rz_endian.h

```diff
@@ -285,6 +307,28 @@ static inline void rz_write_at_le64(void *dest, ut64 val, size_t offset) {
 	rz_write_le64(d, val);
 }
 
+static inline ut128 rz_read_le128(const void *src) {
+	ut128 val = { 0 };
+	val.High = rz_read_at_le64(src, sizeof(ut64));
+	val.Low = rz_read_le64(src);
+	return val;
+}
+
+static inline ut128 rz_read_at_le128(const void *src, size_t offset) {
+	const ut8 *s = ((const ut8 *)src) + offset;
+	return rz_read_le128(s);
+}
+
+static inline void rz_write_le128(void *dest, ut128 val) {
+	rz_write_le64(dest, val.Low);
+	rz_write_at_le64(dest, val.High, sizeof(ut64));
+}
+
+static inline void rz_write_at_le128(void *dest, ut128 val, size_t offset) {
+	ut8 *d = (ut8 *)dest + offset;
+	rz_write_le128(d, val);
+}
```

Comment:
wouldn't make more sense to use RzBitvector ?

Reply:
However, `int128_t` and `uint128_t` are primitive type in many cases.

Reply:
true...